src/services/ErrorService.ts

```diff
@@ -65,6 +65,7 @@ class ErrorService {
       this.logger.warn(message);
     }
 
+    console.error(message);
```

Comment:
remove this

Reply:
This is intended, I'll add a comment explaining why in the code